tests/unit/test_io_convert.py

```diff
@@ -196,6 +217,7 @@ class MixinTestHDF5ToZarr():
                     DirectoryStore('test_export_DirectoryStore.zarr'),
                     TempStore(),
                     NestedDirectoryStore('test_export_NestedDirectoryStore.zarr')]
+    TARGET_FORMAT = "Zarr"
```

Comment:
```suggestion
    TARGET_FORMAT = "ZARR"
```

@mavaylon1  based on the code above, I believe this should be "ZARR" (all uppercase). I think this is likely why the check for references does not appear to be run in the tests.

Reply:
Done